packages/nx/src/command-line/add/add.ts

```diff
@@ -135,8 +135,12 @@ async function initializePlugin(
       args.push(...options.__overrides_unparsed__);
     }
 
-    await runNxAsync(`g ${pkgName}:${initGenerator} ${args.join(' ')}`, {
-      silent: !options.verbose,
```

Comment:
Why this switch? `runNxAsync` handles when the `.nx` Nx installation method is used (non-JS workspaces.

Reply:
It wasn't easy to get prompts working using `runNxAsync`. I'll need to look at it closer.

Reply:
Could switch to `runNxSync` and pass the `stdio` option.